CHANGES.rst

```diff
@@ -8,7 +8,7 @@ Changes to be released in next version
  * 
 
 🐛 Bugfix
- * 
+ * RiotSettings: clear cache resets RiotSettings (#4259).
```

Comment:
Should it be:
```suggestion
 * RiotSettings: Logging out resets RiotSettings (#4259).
```
?

`Clear cache` is another button we have in the settings screen. It does not log the user out.